### PR TITLE
Move census and dashboard into reports

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/controllers/censuses_controller.rb [hunk 1]
```diff
@@ -5,7 +5,8 @@
 ###
 
 class CensusesController < ApplicationController
-  before_action :require_can_view_censuses!, except: [:date_range]
+  include WarehouseReportAuthorization
```
Comment (awisema) [inline app/controllers/censuses_controller.rb:8]: Do we need a migration to grant people with this permission report access?

File: db/seeds.rb [hunk 2]
```diff
@@ -761,6 +761,59 @@ def report_list
         health: false,
       },
     ],
+    'High Level Dashboards' => [
```
Comment (awisema) [inline db/seeds.rb]: We lost some dynamism here vs the old one that itemizes by the installed drivers... Is that OK?

File: app/controllers/censuses_controller.rb (hunk 1 above)
Comment (eanders) [reply app/controllers/censuses_controller.rb:8]: That comes in the next PR https://github.com/greenriver/hmis-warehouse/pull/1316/files

File: db/seeds.rb (hunk 2 above)
Comment (eanders) [reply db/seeds.rb]: Good catch, I'm getting sloppy.  Pushed a fix.

**Pull request merged**